CONTRIBUTING.md

```diff
@@ -58,7 +58,7 @@ When you're ready to contribute, it's time to [Create a pull request](/contribut
 
 #### Contributor License Agreement (CLA)
 
-Before we can accept your pull request, you need to [sign our CLA](https://grafana.com/docs/contribute/cla/). If you haven't, our CLA assistant prompts you to when you create your pull request.
+Before we can accept your pull request, you need to [sign our CLA](https://cla-assistant.io/grafana/grafana). If you haven't, our CLA assistant prompts you to when you create your pull request.
```

Comment:
Are we planning to remove https://grafana.com/docs/grafana/latest/developers/cla/ page or why do we change this link?

Reply:
@torkelo pointed out the link as it was was broken, suggested this fix.

Reply:
That link is referenced multiple times in https://grafana.com/docs/grafana/latest/developers/cla/. @torkelo can you please give some input here. PR been open for 21 days and we should get it merged asap.